llvm/test/TableGen/dag-isel-valuetypelist.td

```diff
@@ -0,0 +1,35 @@
+// RUN: llvm-tblgen -gen-dag-isel -I %p/../../include %s | FileCheck %s
+
```

Comment:
Can you also handle the globalisel emitter

Reply:
Both emitters share CodeGenDAGPatterns.cpp right? So it should just need a test?

Reply:
Yes, probably

Reply:
Added a test for this. As suggested, this didn't require any change to GlobalISel emitter.